live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/repository/filter/TypeFilterApplier.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2023 Alfresco Software Limited
```

Comment:
This is something we should automate. I know we discussed this in the past, but how about if headers were automatically updated for files modified in a commit by pre-commit? This seems uncontroversial and would happen in the same way as our auto-formatting does.

Reply:
I have a spotless fix for that - will introduce in a separate PR.